documentation/src/docs/reader-group-design.md

```diff
@@ -9,85 +9,91 @@ You may obtain a copy of the License at
+Each Reader in a Reader Group is assigned zero or more Segments.
+The Reader assigned to a Segment is the only Reader within the Reader Group that reads Events from that Segment. This is the fundamental mechanism by which Pravega makes ordering guarantees of Event delivery to a Reader – a Reader will receive Events in the order they were written into a Segment.
 There are several challenges associated with this mechanism:
 
-- How to maintain the mapping of which Reader within a ReaderGroup is assigned which Segment
-- How to manage the above mapping when Segments split and merge
-- How to manage the above mapping when Readers are added to the ReaderGroup 
-- How to manage the above mapping when Readers leave the ReaderGroup either by an explicit operation or the Reader becoming unavailable due to network outage or the Reader process failing
+ -  How to maintain the mapping of which Reader within a Reader Group is assigned which Segment?
+ -  How to manage the above mapping when Segments split and merge?
+ -  How to manage the above mapping when Readers are added to the Reader Group?
+ -  How to manage the above mapping when Readers leave the Reader Group either by an explicit operation or the Reader becoming unavailable due to network outage or the Reader process failing?
 
-To solve these problems we can use [StateSynchronizer](state-synchronizer-design.md) to enable readers to coordinate among themselves.
+To address these challenges, we use [State Synchronizer](state-synchronizer-design.md) to enable coordination among Readers.
 
-### How Consistent replicated state can be used to solve the problem
-A consistent replicated state object representing the ReaderGroup metadata will be created in each reader.  This ReaderGroup metadata consists of:
+### Consistent Replicated State
+A consistent replicated state object representing the Reader Group metadata will be created in each Reader. This Reader Group metadata consists of:
 
-- a map of online readers to the segments they own
-- a list of positions in segments that can be taken over.
+ - A map of online Readers to the segments they own.
+ - A list of positions in segments that can be taken over.
 
-Every time the Readers in a ReaderGroup change, the state can be updated. Similarly each time one of the readers is going to start reading from a new segment, it can update the replicated state. 
-This allows all readers to know about all the other Readers in their ReaderGroup and which segments they own.
+Every time the Readers in a Reader Group change, the state can be updated. The replicated state is updated every time when one of the Readers starts reading from a new segment. Thus every Reader is aware of the segments owned by all the Readers in the their Reader Group.
```

Comment:
Please, change: `every time when one of the Readers starts reading from a new segment` -> `every time a Reader starts reading a new Segment.`

Reply:
done